swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+#from unicodedata import category
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition =0.0):
+        super().__init__(condition = condition, category = "Clothing")
```

Comment:
Nice job  removing the category from the constructor, since we don't want to encourage the caller to try to change it. Instead, we pass the literal value that we want to set up to the parent's constructor

With respect to formatting, keep in mind that [PEP8](https://peps.python.org/pep-0008/#other-recommendations) recommends no spaces around `=` when used for default values or keyword arguments. So we might prefer to write

```python
    def __init__(self, condition=0.0):
        super().__init__(condition=condition, category="Clothing")
```